script/build/mk

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+set -e
+
+pkg_topdir=$(cd $(dirname $0)/../../ && pwd -P)
+
+[ "${pkg_os}" = "" ] && pkg_os=`uname -s | tr -s 'A-Z' 'a-z'`
+[ "${pkg_arch}" = "" ] && pkg_arch=`uname -m`
+
+pkg_name=measurement-kit
+
+pkg_dist_dir=${pkg_topdir}/MK_DIST/${pkg_os}/${pkg_arch}/${pkg_name}
+rm -rf ${pkg_dist_dir}
+mkdir -p ${pkg_dist_dir}
+
+pkg_build_dir=${pkg_topdir}/MK_BUILD/${pkg_os}/${pkg_arch}/${pkg_name}
+rm -rf ${pkg_build_dir}
+mkdir -p ${pkg_build_dir}
+
+(
+    set -e
+
+    ./autogen.sh # Must run in the toplevel directory
```

Comment:
Maybe you should add some check to ensure this is actually run in the top level directory (you probably want the same pattern for every build script).